service/resource/legacyv2/adapter/auto_scaling_group.go

```diff
@@ -17,18 +17,18 @@ import (
 type autoScalingGroupAdapter struct {
 	ASGMaxSize             int
 	ASGMinSize             int
-	AZ                     string
+	ClusterID              string
 	HealthCheckGracePeriod int
 	LoadBalancerName       string
 	MaxBatchSize           string
 	MinInstancesInService  string
 	RollingUpdatePauseTime string
-	SubnetID               string
-	ClusterID              string
+	WorkerAZ               string
+	WorkerSubnetID         string
```

Comment:
I only qualified the AZ and Subnet ID as the rest are worker specific.

Reply:
:+1: makes sense